manim/mobject/geometry/arc.py

```diff
@@ -752,12 +770,12 @@ def __init__(
         self,
         label: str | SingleStringMathTex | Text | Tex,
         radius: float | None = None,
-        **kwargs,
+        **kwargs: Any,
     ) -> None:
         if isinstance(label, str):
             from manim import MathTex
 
-            rendered_label = MathTex(label, color=BLACK)
+            rendered_label: VMobject = MathTex(label, color=BLACK)
```

Comment:
Is there a reason for this? It seems to me like `MathTex` would work just fine here.

Reply:
> Is there a reason for this?

The variable `rendered_label` can get its value from either line 778 or 780.
To avoid an assignment type error in line 780 I found a common parent class for `MathTex`, `SingleStringMathTex`, `Text` and `Tex`. This is the `VMobject` used in the type annotation.

Reply:
@JasonGrace2282 Please see comment above.